qiskit_ibm_runtime/utils/noise_learner_result.py

```diff
@@ -34,9 +34,7 @@
 
 from ..utils.embeddings import Embedding
 from ..utils.deprecation import issue_deprecation_msg
-
-if TYPE_CHECKING:
-    import plotly.graph_objs as go
```

Comment:
This looks like a bug that I'm fixing here...`go` will often not exist when it is used below.